CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master / unreleased
 
+## 1.7.0-rc.2
+
+* [BUGFIX] Handle the case of large step causing single pt extents. #3818 #3835
```

Comment:
pt --> point

Reply:
"Handle missing samples due to large steps and single point extents" sounds like a better changelog item though :)

Reply:
we can use that :) I just took the entry from the raw commit history